src/ReleaseHistory.md

```diff
@@ -4,6 +4,7 @@
 * BUGFIX: Fix assertion failed with no clue when TargetFileSpecifiers is null or empty for BinSkim analyze.[763](https://github.com/microsoft/binskim/pull/763)
 * BUGFIX: fix `ERR997.ExceptionLoadingAnalysisTarget : Could not load analysis target` errors analyzing *nix binary resulting from failure to properly parse DWARF debug information.
 * Upgrade `Newtonsoft.JSON` package to 13.0.2 to resolve security alert.
+* Upgrade BinSkim to .net6.0 as .net core 3.1 reached end of support on 12/13/2022.
```

Comment:
v2.0.0-rc2 is released, will we have v2.0.0-rc3?

Reply:
So, rc2 was released so you need to update the release history to reflect this and start the rc3 (Unreleased) section.

Just follow the pattern as you see for rc1